README.md

```diff
@@ -12,100 +12,80 @@
+DLT is for data professionals who use Python to build pipelines.
+- dlt enables non-engineers to build commercial-grade pipelines.
+- dlt minimises maintenance by design.
+- dlt is a better replacement for most home-baked custom pipelines.
+- Community pipeline packages are a good resource for typical needs.
 
-Usually, the source metadata is lacking, so we need to look at the actual data to understand what it is and how to ingest it.
+## Why use DLT?
 
-In order to facilitate this, DLT includes several features
-* Auto-unpack nested json if desired
-* generate an inferred schema with data types and load data as-is for inspection in your warehouse.
-* Use an adjusted schema for follow up loads, to better type and filter your data after visual inspection (this also solves dynamic typing of Pandas dfs)
+DLT is the first python data-pipelining library, meant to be a pip-installable easy to use pipeline creation and deployment kit.
 
-### 2. Safe, scalable loading
+If you end up creating your own pipeline in python, then create it with DLT and benefit from
+- automatic schema inference, deployment, evolution, and data contracts - so you do not have to maintain schemas
+- configurable loading - append, replace, or merge.
+- configurable normalisation engine - decide how to unpack nested documents or specify date formats
+- commercial-grade engineering - dlt implements data engineering best practices such as idempotent, atomic loading.
 
-When we load data, many things can intrrerupt the process, so we want to make sure we can safely retry without generating artefacts in the data.
+## How does it work?
 
-Additionally, it's not uncommon to not know the data size in advance, making it a challenge to match data size to loading infrastructure.
+DLT aims to simplify data loading for everyone.
 
-With good pipelining design, safe loading becomes a non-issue.
+To achieve this, we take into account the progressive steps of data pipelining:
 
-* Idempotency: The data pipeline supports idempotency on load, so no risk of data duplication.
-* Atomicity: The data is either loaded, or not. Partial loading occurs in the s3/storage buffer, which is then fully committed to warehouse/catalogue once finished. If something fails, the buffer is not partially-committed further.
-* Data-size agnostic: By using generators (like incremental downloading) and online storage as a buffer, it can incrementally process sources of any size without running into worker-machine size limitations.
+![](docs/DLT-Diagram_2.png)
+### 1. Data extraction
 
+DLT accepts json or json-producing functions as input, including nested, unstructured data.
 
-### 3. Modelling and analysis
+### 2. Data normalisation
 
-* Instantiate a dbt package with the source schema, enabling you to skip the dbt setup part and go right to SQL modelling.
+DLT features a configurable normalisation engine - it can recursively unpack nested structures into relational tables, or handle various data type conversions.
```

Comment:
```suggestion
DLT features a configurable normalisation engine that recursively unpacks nested structures into relational tables, handles various data type conversions, etc.
```

Reply:
I agree to remove the hyphen but I think we should keep the optional "can" as some folks wanna load raw json string.

Reply:
edited it to add `can` back in :)